scss/_modal.scss

```diff
@@ -236,3 +236,26 @@
 @include media-breakpoint-up(xl) {
   .modal-xl { max-width: $modal-xl; }
 }
+
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-down($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+
+    .modal#{$infix}-fullscreen {
+      width: 100vw;
+      max-width: inherit;
```

Comment:
Why `inherit`?

Reply:
Because .modal-dialog has a max-width:500px and has to be overwritten

Reply:
Use `none` then, that's the default value. `inherit` implies it **needs** to be inherited.

Reply:
@MartijnCuppens  Done